addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19449,7 +19453,7 @@ msgstr ""
 #. Description of setting with label #794 "Allow remote control from programs on other systems"
 #: system/settings/settings.xml
 msgctxt "#36338"
-msgid "Allow programs on the network to control this application."
+msgid "Allow programs on the network to control this application via the JSON-RPC or EventServer protocol, without authentication. This is highly discouraged since it allows anyone with access to the network to completely control this application and, therefore, this deice. Never expose these interfaces on the Internet."
```

Comment:
> If this is "highly discouraged" then running Kodi at all must be highly discouraged. I don't agree with the phrase - otherwise it sounds clear and accurate. This is not a web application, it just intentionally exposes control over HTTP and WS(/raw TCP/ES). 

You can suggest a different phrase but, again, JSONRPC over WS, JSONRPC over TCP, and ES are free-for-all. Having that enabled on a network cannot be good. That is why I chose "discouraged".

> This feature should be improved, not buried.

Surely, but there needs to be someone to do it. And you cannot just change it in core Kodi, but you will have to update all the clients, too.

> Also a misspelling, "deice" to "device".

Thanks!

Reply:
> You can suggest a different phrase

I suggest skipping that phrase, leaving "This allows anyone with access to the network ..."

> Surely, but there needs to be someone to do it. And you cannot just change it in core Kodi, but you will have to update all the clients, too.

I meant only "don't bury it now". Strong verbiage and the "Expert" level change bury it.